ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -4,7 +4,7 @@
 {%     set bgp_vrf = namespace() %}
   vrfs:
 {%     for tenant in network_services_data.tenants %}
-{%         for vrf in tenant.vrfs %}
+{%         for vrf in tenant.vrfs if vrf.name != 'default' %}
```

Comment:
I think we should just adjust the config according to this:
https://www.arista.com/en/support/toi/eos-4-22-0f/14304-evpn-s-irb-in-default-vrf-type-2-type-5-routes

Reply:
Thank you for sharing this doc.

Reply:
I remember now. This is Asymmetric IRB.

Reply:
no, not really.

Reply:
I modified the logic to produce the similar config as explained in the document. I tested it locally.